src/status_im2/contexts/quo_preview/gradient/gradient_cover.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.quo-preview.gradient.gradient-cover
+  (:require [quo2.components.colors.color-picker.view :as color-picker]
+            [quo2.components.gradient.gradient-cover.view :as gradient-cover]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.blur :as blur]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Color:"
+    :key     :color
+    :type    :select
+    :options (mapv (fn [color]
+                     {:key color :value color})
+                   color-picker/color-list)}
+   {:label "Blur (dark only)?"
+    :key   :blur?
+    :type  :boolean}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:color :blue :blur? false})
+        blur? (reagent/cursor state [:blur?])
+        color (reagent/cursor state [:color])]
+    [:f>
+     (fn []
+       (rn/use-effect (fn []
+                        (when @blur?
+                          (quo.theme/set-theme :dark)))
+                      [@blur?])
+       [:<>
+        [preview/customizer state descriptor]
+        [rn/view
+         {:style {:height        332
+                  :margin-top    24
+                  :overflow      :hidden
+                  :border-radius 12}}
+         (when @blur?
+           [rn/image
+            {:style {:height 332}
+             :source
+             {:uri
+              "https://4kwallpapers.com/images/wallpapers/giau-pass-mountain-pass-italy-dolomites-landscape-mountain-750x1334-4282.jpg"}}])
```

Comment:
I think we shouldn't use external links like that. Links can change. It is better to add the file. I have added that file in a PR https://github.com/status-im/status-mobile/pull/16719/files#diff-f3617f44ee14ed3a0105c26ac10aaddba11673c63caa9f29fbb4afbce2411f73

Reply:
Thanks, @OmarBasem!

I started with the blur-view in `status-im2.contexts.quo-preview.preview` ns, but it doesn't play well. So, I ended up adding the blur in this same preview file. I have to refactor that `preview/blur-view`. But, I will do it separately and also add this image to it.